src/payments/dtos/payment-report.dto.ts

```
@@ -30,6 +30,13 @@ export class PaymentReportItemDto {
 
   @ApiProperty({ type: 'string', description: 'Payment status (PAID, FAILED, PENDING)' })
   status: string;
+
+  @ApiProperty({
+    type: 'boolean',
```

ESLint

[error] Replace `'boolean'` with `"boolean"`

(prettier/prettier)